src/battlefield.rs

```diff
@@ -1,3 +1,4 @@
+// REVIEW: unimportant, but generally people start from zero.

```

Comment:
The idea was that inside Option a value of this type is represented by zero, so going from `None::<Figure>` to `None::<Mark>` and vice-versa is a no-op. Possibly just a premature optimisation. Since these values are not actually used in code, is seems a good idea to remove them completely